core/controllers/access_validators_test.py

```diff
@@ -826,6 +828,54 @@ def test_should_not_access_for_logged_out_user(self) -> None:
         )
 
 
+class ExplorationEditorPageAccessValidationHandlerTests(
+    test_utils.GenericTestBase):
+    """Checks the access to the exploration editor page and its rendering."""
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.guest_username = 'guest'
+        self.guest_email = 'guest@example.com'
+        self.signup(self.guest_email, self.guest_username)
+        self.owner_id = self.get_user_id_from_email(self.guest_email)
+        self.exp_id = 'unpub_eid'
+        exploration = exp_domain.Exploration.create_default_exploration(
+            self.exp_id)
+        exp_services.save_new_exploration(self.owner_id, exploration)
+
+    def test_access_exploration_editor_page_without_logging_in(self) -> None:
+        self.get_html_response(
+            '%s/can_access_exploration_editor_page/%s' % (
+                ACCESS_VALIDATION_HANDLER_PREFIX, self.exp_id
+            ), expected_status_int=404
+        )
+
+    def test_access_exploration_editor_page_after_logging_in(self) -> None:
+        self.login(self.guest_email)
+        self.get_html_response(
+            '%s/can_access_exploration_editor_page/%s' % (
+                ACCESS_VALIDATION_HANDLER_PREFIX, self.exp_id
+                ), expected_status_int=200
```

Comment:
Deindent by 4 to match scope of corresponding opening paren

Reply:
Done